src/main/java/competition/subsystems/drive/commands/DriveToPositionCommand.java

```diff
@@ -37,15 +53,21 @@ public void execute() {
 
         // How you do this is up to you. If you get stuck, ask a mentor or student for
         // some hints!
-        drive.tankDrive(0.25,0.25);
-        pose.getPosition();
+        this.error = targetPosition - pose.getPosition();
+        this.speed = pose.getPosition() - previousPosition;
+        double fattyPower = error / 2.5 - speed * 12;
```

Comment:
⭐ ⭐ ⭐  you're not using your pProperty, should this instead be?
```suggestion
        double fattyPower = error / pProperty.get() - speed * 12;